### ISSUE TITLE
fails installation

### ISSUE THREAD

**Issue by NikolaMandic** [senny/cabbage#228]
fails when cloning https://github.com/emacsmirror/mode-compile/

**Comment by jiegec** [comment]
I got a 404 error when opening that url.

**Comment by NikolaMandic** [comment]
I tried to install cabbage and got this if you do not get it then I guess it is my machine

**Comment by jiegec** [comment]
Yes that url is included in the .gitmodules. But the project hasn't been updated for a long time.

**Comment by NikolaMandic** [comment]
oh I did not notice. Thanks
